doc/tutorials/notifications.md

```diff
@@ -257,6 +257,116 @@ int main (void)
 }
 ```
 
+### How-To: Reload KDB when Elektra's configuration has changed
+
+This section shows how the notification feature can be used to reload an
+application's KDB instance when Elektra's configuration has changed.
+This enables applications to apply changes to mount points or globally mounted
+plugins without restarting.
+
+**Step 1: Register for changes to Elektra's configuration**
+
+To achieve reloading on Elektra configuration changes we register for changes
+below the key `system/elektra` using
```

Comment:
Does `/elektra` works too? It might be necessary once we have #1074

Reply:
Yes, `/elektra` works too. I'll update the example.